devlib/target.py

```diff
@@ -1156,6 +1156,9 @@ def dump_logcat(self, filepath, filter=None, append=False, timeout=30):  # pylin
     def clear_logcat(self):
         adb_command(self.adb_name, 'logcat -c', timeout=30)
 
+    def logcat_monitor(self, regexps=['.*']):
```

Comment:
`get_logcat_monitor` or `monitor_logcat` -- method names should generally be verb phrases.